
int32 left_pwm
int32 right_pwm
bool left_dir
bool right_dir

